LayoutTests/fast/forms/auto-fill-button/input-loading-auto-fill-button.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body>
+<p>This tests that the Loading AutoFill button renders. It can only be tested in the test tool.</p>
+<div id="container">
```

Comment:
NIT: Is this necessary, or can we just have the `<input type="text">` be toplevel?

Reply:
I'm not sure if its necessary; I just followed the convention of the other autofill test HTML files

Reply:
I think you can remove it.